src/ducks/apps/components/ApplicationPage/Details.jsx

```diff
@@ -48,12 +52,14 @@ export const Details = ({ app, description, changes, parent, mobileApps }) => {
     !!categories.length &&
     categories.map(c => t(`app_categories.${c}`))
   const developerName =
-    developer && getTranslatedManifestProperty(app, 'developer.name', t)
+    (developer && getTranslatedManifestProperty(app, 'developer.name', t)) ||
+    (isShortcutFile(app) && app.attributes?.metadata?.source)
   const developerUrl =
-    developer && getTranslatedManifestProperty(app, 'developer.url', t)
+    (developer && getTranslatedManifestProperty(app, 'developer.url', t)) ||
+    app.attributes?.metadata?.url
   const shortVersion = version && version.match(/^(\d+\.\d+\.\d+)-.*$/)
   const displayedVersion =
-    (shortVersion && shortVersion.length && shortVersion[1]) || version
+    ((shortVersion && shortVersion.length && shortVersion[1]) || version) ?? '-' // If no version, display a dash
```

Comment:
just to be sure to understand, this is related to the updated snapshots right? as in, they shouldn't have been updated since the tests don't have shortcut data